src/main/java/com/googlecode/jmeter/plugins/webdriver/config/gui/ChromeDriverConfigGui.java

```diff
@@ -22,11 +22,6 @@ public String getStaticLabel() {
 		return JMeterPluginsUtils.prefixLabel("Chrome Driver Config");
 	}
 
-	@Override
-	protected String getWikiPage() {
-		return "DirectDriverConfig";
```

Comment:
AFAIK if you set address of doc page with "https://" in the beginning, it should display correct link in UI. If that is what you actually wanted

Reply:
This is what I have done in WebDriverConfigGUI:
public static final String WIKIPAGE = "https://github.com/undera/jmeter-plugins-webdriver";
	public String getWikiPage() {
		return WIKIPAGE;
	}

and have removed all old urls as ther eis only one and unique wiki page which is the same for alll browsers.